x-pack/plugins/rule_registry/server/plugin.ts

```diff
@@ -95,6 +104,22 @@ export class RuleRegistryPlugin
 
     this.ruleDataService.initializeService();
 
+    core.getStartServices().then(([_, depsStart]) => {
+      const ruleRegistrySearchStrategy = ruleRegistrySearchStrategyProvider(
+        depsStart.data,
+        this.ruleDataService!,
+        depsStart.alerting,
+        logger,
+        plugins.security,
+        depsStart.spaces
+      );
+
+      plugins.data.search.registerSearchStrategy(
+        'ruleRegistrySearchStrategy',
```

Comment:
Just to make sure that people are aware is about alerts
```suggestion
        'ruleRegistryAlertsSearchStrategy',
```

Reply:
Done!